### ISSUE TITLE
Unnecessary local variable `dn` in twisted.internet._sslverify.KeyPair.signRequestObject

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6498]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#6498|
|Type|defect|
|Created|2013-04-30 00:19:59Z|

The line:

```
   dn = requestObject.getSubject()
```

is not necessary.  The method call has no important side-effects and the local variable is not by any following code.

Attachments:

* [ticket6498.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3ac/3ac2069318a1039378ec477737553cbe9096949b/863795937f502bde3af1f503c58c4852b47e0a08.patch) (638 bytes) - added by canibanoglu on 2013-04-30 18:55:55Z - removed the unused line, added a .misc topfile

<details><summary>Searchable metadata</summary>

```
trac-id__6498 6498
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1367281199000000 1367281199000000
changetime__1370047067000000 1370047067000000
version__None None
owner__therve therve

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="canibanoglu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|canibanoglu set owner to canibanoglu|
|-|-|

Just one question before I submit a patch for this, should this one be deleted right away or should it be deprecated and then deleted at a later point in time? I believe it's the former but it's always nice to be sure

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

> Just one question before I submit a patch for this, should this one be deleted right away or should it be deprecated and then deleted at a later point in time? I believe it's the former but it's always nice to be sure

Yes, this can just be deleted.  There's no need for a deprecation because no public part of the interface (either the signature of a method, the allowed argument values for any of the parameters, or some sufficiently important aspect of the implemented behavior) is being changed.  Only an internal implementation detail is changing.

Thanks!

**Comment by twisted-trac** [comment]
|<img alt="canibanoglu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|canibanoglu removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set owner to @therve|
|-|-|

Looks good, applying.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set status to `closed`|
|-|-|

(In [38578]) Remove a unused variable in sslverify KeyPair.signRequestObject method. 

Author: canibanoglu
Reviewer: therve
Fixes: [#6498](https://github.com/twisted/twisted/issues/6498)
